service/controller/aws/v17/version_bundle.go

```diff
@@ -7,6 +7,11 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "cluster-autoscaler",
+				Description: "Update to 1.14.3. More info here: https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.14.3",
```

Comment:
```suggestion
				Description: "Update to 1.14.3. https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.14.3",
```

Reply:
Yes, I like that too.  Makes changelog simpler.